sources/packages/backend/workflow/Dockerfile.dev

```diff
@@ -0,0 +1,26 @@
+# Base Image
+FROM node:16.17.0-alpine3.16
+
+LABEL maintainer="BC GOV"
+
+# Updating container path variable with node bin
+ENV PATH $PATH:/app/node_modules/.bin
+
+WORKDIR /app
+
+# Copying sources
+COPY libs ./libs
+COPY workflow ./workflow
+COPY ./*.json ./
+COPY ./*.js ./
+
+# Installing dependencies
+RUN npm ci
+
+# Adding a nonroot user
```

Comment:
for my understanding. why is this for?

Reply:
@dheepak-aot is this the Sonarcloud rule that triggered the change?
https://rules.sonarsource.com/docker/RSPEC-6471

Reply:
Yes @andrewsignori-aot 👍. Thanks for sharing it. @ann-aot sonar finds running the container as root user as security-sensitive